samples/SmokeTest/SmokeTest/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Azure.Messaging.EventHubs;
+using System;
+using System.Threading.Tasks;
+
+namespace SmokeTest
+{
+
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            Console.WriteLine("SMOKE TEST FOR TRACK 2 LIBRARIES");
+            await KeyVaultTest.performFunctionalities();
+            await BlobStorage.performFunctionalities();
```

Comment:
`KeyVaultTest` is a great name. We should standardize the names of these classes so it's clear that their goals are similar. For example `BlobStorage` should be `BlobStorageTest` and `EventHubs` should be `EventHubsTest`